.github/workflows/percy-demo-tests.yml

```diff
@@ -52,6 +60,8 @@ jobs:
         run: NODE_OPTIONS=--max_old_space_size=4096 npm install 
 
       - name: Cypress run test percy
+        if: github.event.inputs.testsKind != "functional"
```

Comment:
If we run it via workflow dispatch github.event.inputs.testsKind will be "undefined" and in this case I want to run all tests. I was thinking about adding new step which is adding new environment variable and when environment is trigger via workflow dispatch it will be setting this env as, "all", and when it is run via manual click it can set this variable from github.event.inputs.testsKind. But I'm not sure if it just wasn't be complicated and lengthen code

Reply:
Ok if it's also triggered via repository-dispatch, then I'd suggest to use also `github.event.client_payload.testsKind` as a call to point out the type of the testing
[Documentation about the feature](https://docs.github.com/en/rest/repos/repos?apiVersion=2022-11-28#create-a-repository-dispatch-event)

Reply:
Done, thanks!
Resolving